Kernel/Bus/PCI/DeviceTreeInitializer.cpp

```diff
@@ -32,26 +32,22 @@ void initialize()
     // [2]: https://github.com/devicetree-org/dt-schema/blob/main/dtschema/schemas/pci/pci-bus-common.yaml
     // [3]: https://github.com/devicetree-org/dt-schema/blob/main/dtschema/schemas/pci/pci-host-bridge.yaml
 
-    // The pci controllers are usually in /soc/pcie?@XXXXXXXX
-    // FIXME: They can also appear in the root node, or any simple-bus other than soc
+    // The pci controllers are usually in /soc/pcie?@XXXXXXXX on RISC-V, and in /pcie?@XXXXXXXX on AArch64
+    // FIXME: They can also appear in any simple-bus other than soc
     auto const& device_tree = DeviceTree::get();
 
     auto maybe_soc = device_tree.get_child("soc"sv);
-    if (!maybe_soc.has_value()) {
-        dmesgln("PCI: No `soc` node found in the device tree, PCI initialization will be skipped");
-        return;
-    }
 
-    auto const& soc = maybe_soc.value();
+    auto const& pci_root_node_parent = maybe_soc.has_value() ? maybe_soc.value() : device_tree;
```

Comment:
Ah no I meant calling this node something like `pci_node_parent` or `pci_host_controller_parent`.